python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -410,13 +410,18 @@ def optimize(self, model: nn.Module,
                     val_dataset = RepeatDataset(sample=(val_sample, val_label), num=1)
                     validation_data = DataLoader(val_dataset, batch_size=1)
                     validation_data = remove_batch_dim_fn(validation_data)
-
+        # jit cannot handle `Mapping`, so we convert it to `dict`
+        if isinstance(input_sample, (Mapping, MappingView)):
```

Comment:
https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes
Maybe MappingView cannot convert to dict